safe-global/web-core
# fix: add `shortName` attribute to Beamer user

## What it solves

Allows for Beamer top bar announcements

## How this PR fixes it

A custom `chain` attribute has been added to Beamer users which we can leverage as a filter. Each `chain` has been added to "segment filters" on Beamer, ready for top bar announcements as follows:

![image](https://user-images.githubusercontent.com/20442784/193255061-e38bdfe4-7606-4d9e-affd-a849a62a8bea.png)

The colours of Beamer have been updated to match the new theme. The top bar is set to that of `info.main` as it best matches light/dark mode. All colours can be easily adjusted via the Beamer configuration panel, however.

## How to test it

1. Open the Safe NOT on Rinkeby (do so via the URL so that it doesn't default to Rinkeby). Observe no top bar.
2. Switch to Rinkeby and observe the new top bar.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/193255290-bd643f24-62ba-4888-a17d-8b3ee2a4a335.png)

![image](https://user-images.githubusercontent.com/20442784/193255310-add3845d-7eba-487c-aa05-eaffd0a27d0f.png)

![image](https://user-images.githubusercontent.com/20442784/193255329-fbd52806-9f86-4c40-9461-37a1b211efe5.png)

![image](https://user-images.githubusercontent.com/20442784/193255342-3f7c868b-22bd-4d63-b629-2a4ae48b38c5.png)
